admin/resources/js/components/H2Components/Pill.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Pill: React.FC<{ content: string }> = ({ content }) => {
+  return (
+    <span
+      data-h2-border="b(darkpurple, all, solid, s)"
+      data-h2-display="b(block)"
+      data-h2-radius="b(m)"
+      data-h2-bg-color="b(lightpurple[.1])"
+      data-h2-padding="b(all, s)"
+      data-h2-color="b(darkpurple)"
+      data-h2-font-size="b(caption)"
+      role="cell"
```

Comment:
Will the Pill component be nested in an element with the role of row? If not, maybe the role value should be a prop?

> To be supported, the cell must be nested in an element with the role of row.
source: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Cell_Role

Reply:
Similarly, colour should be controllable via a prop. At least within several options (like the button). Project design docs suggest we should have 4 versions for this: purple-solid, purple-outline, navy-blue-solid, and navy-blue-outline:
https://www.figma.com/file/tXs6IdvpuX4jCLBWbJx0C6/Components-and-Patterns?node-id=34%3A3398

Reply:
It seems to call these "chips" there, btw